### PR TITLE
[7] Develop a contact form page using HTML

### PR DESCRIPTION
**Pull request by Chidiebere-debug**
## Task: `Contact-form'
Screenshot (27)

### CODE REVIEW

**Comment by ToluAkin** [comment]
@Chidiebere-debug The output that was required of you is the view on the web

File: submissions/onyemesimChidiebereSabinus/html2/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0808/160401.932:ERROR:settings.cc(401)] Settings magic is not 1129342067
```
Comment (ToluAkin) [inline submissions/onyemesimChidiebereSabinus/html2/debug.log:1]: Remove this file @Chidiebere-debug

File: submissions/onyemesimChidiebereSabinus/html2/contact.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <label for="fullname">Full Name</label><br>
+            <input type="text" name="fullname" placeholder="surname firstname middlename"><br><br>
+            <label for="email">Email</label><br>
+            <input type="text" name="email" ><br><br>
+            <label for="complaint">Subject</label><br>
+            <textarea id="complaint" name="complaint" rows="4" cols="30"></textarea><br>
+            <input type="Submit"><br><br>
+            <em>Or</em><br>
+            <p>Send us a message via:</p> 
+            <a href="#"><img src="twitter.jpg" alt="twitter logo" width="30" height="30"></a>
```
Comment (ToluAkin) [inline submissions/onyemesimChidiebereSabinus/html2/contact.html:21]: Remove `width` and `height` attributes

File: submissions/famuyiwaOlawumi/html/index.html [hunk 3]
```diff
@@ -8,14 +8,11 @@
...
         <h1>XYZ MART</h1>
     </a>
-    <nav>
-        <a href="index.html">Product page</a>
-        <a href="../html-forms/contactform.html">Contact Us</a>
-    </nav>
     <ul>
         <li><a href="#">My cart</a></li>
-        <li><a href="#">Login</a></li>
+        <li><a href="#">Login/Singup</a></li>
```
Comment (ToluAkin) [inline submissions/famuyiwaOlawumi/html/index.html:13]: There is a typo here

File: submissions/onyemesimChidiebereSabinus/html2/contact.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/onyemesimChidiebereSabinus/html2/contact.html:1]: This file should be in a folder html-form @Chidiebere-debug

**Comment by adaugochi** [comment]
You have to update your branch. Reach out to me when you are available

**Comment by Chidiebere-debug** [comment]
Goodevening.I will be avaliable tomorrow to rectify this with your direction.  Fix a time and I will make myself avalaible
    On Monday, August 17, 2020, 02:23:23 AM PDT, Maryfaith Mgbede <notifications@github.com> wrote:  
 
 


You have to update your branch. Reach out to me when you are available

**Comment by adaugochi** [comment]
Long over due

**Pull request closed without merging**